crowdfunding_compassion/views/project_donation_page.xml

```diff
@@ -0,0 +1,128 @@
+                                        <span class="donation-type__card-icon"/>
+                                        <h4 class="blue text-center uppercase">Sponsor children</h4>
+                                        <h6 class="blue text-center uppercase">(You will be redirected to compassion switzerland website)</h6>
+
+                                        <!-- Show the select button also if there is a fund option-->
+                                        <t t-if="project.product_id">
+                                            <span class="fake-btn my-3">SELECT</span>
+                                        </t>
+                                    </div>
+                                </label>
+                            </div>
+
+                            <!-- Project impact card without progress bars -->
+                            <div class="col-sm-12 col-md-4 mb-3">
+                                <t t-set="display_impact" t-value="False" />
+                                <t t-call="crowdfunding_compassion.project_impact_card" />
+                            </div>
+                        </div>
+                    </div>
+                </section>
+
+                <!-- Participant selection -->
+                <section>
+                    <div class="container">
+                        <h3 class="blue uppercase mt-5 mb-3">I support the project of the following fundraiser</h3>
+
+                        <div>
+                            <t t-foreach="project.participant_ids" t-as="participant">
+                                <label>
+                                    <!-- If the query parameter [participant_id] matches the currently displayed participant, check it -->
+                                    <t t-if="selected_participant and int(selected_participant) == participant.id">
+                                        <input type="radio" name="participant" class="card-input-element d-none" t-att-value="participant.id" checked="checked"/>
+                                    </t>
+                                    <t t-else="">
+                                        <input type="radio" name="participant" class="card-input-element d-none" t-att-value="participant.id"/>
+                                    </t>
+                                    <t t-call="crowdfunding_compassion.participant_picture" />
+                                </label>
+                            </t>
+                        </div>
+                        <a id="url" href="">
+                            <button id="submit" type="submit" class="btn btn-primary btn-lg mb-5 float-right uppercase">Next step</button>
+                        </a>
+                    </div>
+                </section>
+
+                <!-- This function disables the next button if there isn't 2 radios checked
+                (fund/sponsor and participant) and adjusts the link for next step -->
+                <script>
```

Comment:
Could we move this as well into a separate js file?

Reply:
Will do